packages/service/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "iotsec-service",
+  "version": "1.0.0",
+  "description": "An IOT security system using a Particle Photon and a Raspberry Pi",
+	"type": "module",
+  "exports": "./dist/program.js",
+	"engines": {
+		"node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
I'm ok with limiting this to v16 to prevent package-lock.json conflicts. Apparently v16 uses `lockfileVersion` 2 but v14 uses `lockfileVersion` 1, and going from one to the other introduces a ton of extraneous changes.